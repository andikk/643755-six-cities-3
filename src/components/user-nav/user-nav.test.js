import React from "react";
import renderer from "react-test-renderer";
import UserNav from "./user-nav.jsx";

it(`Should UserNav render correctly`, () => {

  const tree = renderer
    .create(<UserNav
      authorizationStatus = {`NO_AUTH`}
      user={{}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
