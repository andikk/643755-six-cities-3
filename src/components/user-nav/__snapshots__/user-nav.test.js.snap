// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should UserNav render correctly 1`] = `
<nav
  className="header__nav"
>
  <ul
    className="header__nav-list"
  >
    <li
      className="header__nav-item user"
    >
      <a
        className="header__nav-link header__nav-link--profile"
        href="/favorites"
        onClick={[Function]}
      >
        <div
          className="header__avatar-wrapper user__avatar-wrapper"
        >
           
        </div>
        <span
          className="header__user-name user__name"
        />
      </a>
    </li>
  </ul>
</nav>
`;
